### DISCUSSION TITLE
A metal pole

### DISCUSSION THREAD

**Discussion by Mamuts123** [FakeFishGames/Barotrauma#12314] [Ideas and suggestions]
[NEW DEVELOPMENT IDEA] [SUBMARINE PART] [SUBMARIME EDITOR] This could be attached anywhere on the sub (altough it should be attached on the outside) and would allow a player to hold on to it. This uses one hand but makes is impossible to fall over from crashes or high speeds. If put on to the outside of the sub / a drone, you will hold on and be able to basically ride that vessel.  It's main purpose would be for mission where you explore caves with a small un-manned drone like the kastrull's, so crew-members can swim with it. Downside is you only have one hand free.

**Comment by Hazmat616** [comment]
Perhaps also allow safety harness's and dice suits to anchor themselves to surfaces and connects them in a similar way to harpoon ropes.
